prompts/lint/100_user_prompt.md

```diff
@@ -0,0 +1,12 @@
+MY project has the following languages used:
+
+{{#project.files}}
```

Comment:
Does this mean that you think using the files directly is an improvement over using linguist?